src/components/learner-credit-management/members-tab/MemberStatusTableCell.jsx

```diff
@@ -19,26 +21,75 @@ const MemberStatusTableCell = ({
   let popoverExtra2;
   if (row.original.status === 'pending') {
     icon = Timelapse;
-    text = 'Waiting for member';
-    popoverHeader = `Waiting for ${row.original.memberDetails.userEmail}`;
-    popoverBody = 'This member must accept their invitation to browse this budget\'s catalog '
-    + 'and enroll using their member permissions by logging in or creating an account within 90 days.';
-    popoverExtra1 = 'Need help?';
-    popoverExtra2 = 'Learn more about adding budget members in Learner Credit at ';
+    text = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.pending',
+      defaultMessage: 'Waiting for member',
+      description: 'Status of the member invitation',
+    });
+    popoverHeader = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.pendingPopoverHeader',
+      defaultMessage: 'Waiting for {userEmail}',
+      description: 'Popover header for the pending status',
+    }, { userEmail: row.original.memberDetails.userEmail });
+    popoverBody = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.pendingPopoverBody',
+      defaultMessage: "This member must accept their invitation to browse this budget's catalog and enroll using their member permissions by logging in or creating an account within 90 days.",
```

Comment:
Can we use apostrophe as value here?

Reply:
I think we don't need to make the apostrophe variable because it may not exist in other languages. That's the reason for not making it a variable.